.gitignore

```diff
@@ -2,6 +2,7 @@ _site/
 .sass-cache/
 .jekyll-cache/
 .jekyll-metadata
+_blogs/
```

Comment:
We don't have to add this folder to .gitignore, since we do really need to add the content in that folder.

If what you are trying to do is that the files you create are not added to the repository, you can edit the `.git/info/exclude` file and add it there, it won't be commited to the repository and you won't see it inside git.